### DISCUSSION TITLE
Exposing additional modbus registers

### DISCUSSION THREAD

**Discussion by JenertsA** [gracingpro/blauberg-homeassistant#5] [Q&A] [answered]
Hi, 

Played around with the solution and it is great! 
Works as expected and helped to integrate my Blauberg device into my smart home ecosystem quickly and easily. 

Additionally, I would like to gather and control a bit more settings. Bypass is one of the things I would like to control. 
I would like to expose `HR_BPS_ROTOR_MODE` register with the address `74`. 

Is there a particular reason that not all S21 controller registers are exposed? 
I get the general idea of how to expose additional addresses, but do I need to expose everything up to `74` or can I do some subset on only that one? 

Thanks for the pointers and ideas!

**Comment by gracingpro** [comment, marked as answer]
Hi there,

Thank you for your positive feedback on the solution! It's great to hear that it's been effective in integrating your Blauberg device into your smart home ecosystem.

You can certainly expose a specific register like the one at address 74 without needing to expose everything up to that point. The system is designed to allow for selective inclusion of registers based on user needs.

However, I must inform you that development is currently on pause as I don't have access to a Blauberg device to test and validate changes. This limits my ability to implement and support new features at the moment. I am open to community contributions, so if you or anyone else in the community is willing to test and contribute to the development, it would be greatly appreciated.

Thank you again for your interest and understanding. I look forward to any future collaborations!
